python/bigipconfigdriver.py

```diff
@@ -366,10 +353,10 @@ def create_config_kubernetes(partition, config):
     f5 = {'ltm': {}, 'network': {}}
     if 'openshift-sdn' in config:
         f5['network'] = create_network_config_kubernetes(config)
-    if 'resources' in config and 'virtualServers' in config['resources']:
-        f5['ltm'] = create_ltm_config_kubernetes(partition,
-                                                 config['resources'])
+    if 'resources' in config and partition in config['resources']:
```

Comment:
What this ends up doing is stamping 'network' for every partition running extra apply fdb configs. What would be better here is to keep ltm and network separate.

In do reset we can just loop over managers and get the partition config. Then after that loop do one single network call.

Reply:
Agree, good catch!